### PR TITLE
memcache quality of life improvements

### PR DESCRIPTION
**Pull request by bsimpson63**
👓 @spladug @pacejackson

### CODE REVIEW

**Comment by bsimpson63** [comment]
Ah, actually the version of pymemcache we're on (1.3.2) doesn't support default_noreply and instead all the operations default to `noreply=True`. The argument was added in version 1.3.3 https://github.com/pinterest/pymemcache/commit/ca9d579b722198de6188695afb63aa26689ecc71

**Comment by bsimpson63** [comment]
💇 We could either upgrade pymemcache or make a simple wrapper around the client. I don't want to deal with packaging since there's no ubuntu package to backport.

**Comment by pacejackson** [comment]
How do you feel about making the wrapper take the extra constructor argument and do what the updated client does rather than just making the default value `False`?

File: baseplate/context/memcache/__init__.py [hunk 1]
```diff
@@ -9,9 +9,40 @@
...
+        return super(OptInNoReplyClient, self).delete_many(keys, noreply)
+
+    def incr(self, key, value, noreply=False):
+        return super(OptInNoReplyClient, self).incr(key, value, noreply)
+
+
 def pool_from_config(app_config, prefix="memcache.", serializer=None,
                      deserializer=None):
-    """Make a PooledClient from a configuration dictionary.
+    """Make a OptInNoReplyClient from a configuration dictionary.
```
Comment (pacejackson) [inline baseplate/context/memcache/__init__.py]: "Make an

**Comment by spladug** [comment]
It's a pure python package, I don't think packaging will be that hard. It feels pretty unpleasant to have to add a bunch of stuff here just because we're using an out-of-date library. :nail_care:

**Comment by bsimpson63** [comment]
this is old, closing

**Pull request closed without merging**